tsl/test/sql/.gitignore

```diff
@@ -5,3 +5,8 @@
 /ordered_append-*.sql
 /plan_skip_scan-*.sql
 /transparent_decompression*-1[3-9].sql
+/chunk_utils_internal-*.sql
+/compress_bloom_sparse-*.sql
```

Comment:
Can we suppress the "shell type" notice with the client_min_messages like you did for another test?

Reply:
Leaving it as is.